### ISSUE TITLE
bower-asset/jquery cannot be satisfied

### ISSUE THREAD

**Issue by ethanclevenger91** [yiisoft/yii2#11398]
### What steps will reproduce the problem?

I wager the following composer.json and running a `composer install`:

```
{
    "name": "yiisoft/yii2-app-basic",
    "description": "Yii 2 Basic Project Template",
    "keywords": ["yii2", "framework", "basic", "project template"],
    "homepage": "http://www.yiiframework.com/",
    "type": "project",
    "license": "BSD-3-Clause",
    "support": {
        "issues": "https://github.com/yiisoft/yii2/issues?state=open",
        "forum": "http://www.yiiframework.com/forum/",
        "wiki": "http://www.yiiframework.com/wiki/",
        "irc": "irc://irc.freenode.net/yii",
        "source": "https://github.com/yiisoft/yii2"
    },
    "minimum-stability": "dev",
    "require": {
        "php": ">=5.4.0",
        "yiisoft/yii2": "^2.0.5",
        "yiisoft/yii2-bootstrap": "*",
        "yiisoft/yii2-swiftmailer": "*",
        "dektrium/yii2-user": "0.9.*@dev",
        "kartik-v/yii2-widgets": "*",
        "dektrium/yii2-rbac": "dev-master",
        "synatree/yii2-dynamic-relations": "dev-master"
    },
    "require-dev": {
        "yiisoft/yii2-codeception": "*",
        "yiisoft/yii2-debug": "*",
        "yiisoft/yii2-gii": "*",
        "yiisoft/yii2-faker": "*"
    },
    "config": {
        "process-timeout": 1800
    },
    "scripts": {
        "post-create-project-cmd": [
            "yii\\composer\\Installer::postCreateProject"
        ]
    },
    "extra": {
        "yii\\composer\\Installer::postCreateProject": {
            "setPermission": [
                {
                    "runtime": "0777",
                    "web/assets": "0777",
                    "yii": "0755"
                }
            ],
            "generateCookieValidationKey": [
                "config/web.php"
            ]
        },
        "asset-installer-paths": {
            "npm-asset-library": "vendor/npm",
            "bower-asset-library": "vendor/bower"
        }
    }
}
```

### What is the expected result?
Update composer packages

### What do you get instead?
```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - yiisoft/yii2 2.0.5 requires bower-asset/jquery 2.1.*@stable | 1.11.*@stable -> no matching package found.
    - yiisoft/yii2 2.0.x-dev requires bower-asset/jquery.inputmask ~3.2.2 -> no matching package found.
    - yiisoft/yii2 2.0.7 requires bower-asset/jquery.inputmask ~3.2.2 -> no matching package found.
    - yiisoft/yii2 2.0.6 requires bower-asset/jquery 2.1.*@stable | 1.11.*@stable -> no matching package found.
    - yiisoft/yii2 2.0.7 requires bower-asset/jquery.inputmask ~3.2.2 -> no matching package found.
    - Installation request for yiisoft/yii2 ^2.0.5 -> satisfiable by yiisoft/yii2[2.0.5, 2.0.6, 2.0.7, 2.0.x-dev].

Potential causes:
 - A typo in the package name
 - The package is not available in a stable-enough version according to your minimum-stability setting
   see <https://getcomposer.org/doc/04-schema.md#minimum-stability> for more details.

Read <https://getcomposer.org/doc/articles/troubleshooting.md> for further common problems.
```

### Additional info

| Q                | A
| ---------------- | ---
| Yii version      |  Attempting to upgrade to 2.0.7
| PHP version      |  7.0.5
| Operating system |  Windows 10
| fxp/composer-asset-plugin | 1.1.3

I think this may have something to do with the latest update to the fxp/composer-asset-plugin, but that's a stab.

**Comment by KasianenkoMykhailo** [comment]
Try this, it might help.
Run in console
composer global require "fxp/composer-asset-plugin:~1.1.1"

**Comment by ethanclevenger91** [comment]
That version identifier still resolves to 1.1.3, so it doesn't install anything new.

**Comment by ethanclevenger91** [comment]
Welp it seems to have resolved itself after trying to require a new library. Closing for now.
